src/noun.substance

```diff
@@ -4823,9 +4838,12 @@ hypo: chemical_compound
 
 w: bitter_spar
 w: dolomite[1] drf adj.pert:dolomitic
-d: a light colored mineral consisting of calcium magnesium carbonate; a source of magnesium; used as a ceramic and as fertilizer
+d: a light colored mineral consisting of calcium magnesium carbonate (CaMg(CO3)2); a source of magnesium; used as a ceramic and as fertilizer
```

Comment:
I would avoid the formula.

Reply:
Formulas are used elsewhere in WN, thought it might improve the definition. I’d rather keep it, but won’t make a big deal out of it.

Reply:
not so many... You are adding systematically.

Reply:
Examples: calcium_hydride; calcium_carbide; calcium_sulfate; carbonyl_group; carboxyl; and the list goes on

Reply:
Ah, OK.